inc/class_error.php

```diff
@@ -539,6 +539,28 @@ function output_error($type, $message, $file, $line)
 			$charset = 'UTF-8';
 		}
 
+		if(isset($mybb->user) && is_array($mybb->usergroup) && $mybb->usergroup['cancp'] == 1)
+		{
+			// admins directed to contact MyBB
+			$contact = 'Please contact the <a href="https://mybb.com">MyBB Group</a> for technical support.';
+		}
+		else
+		{
+			// users directed to contact the site owner
+			$contact_us = 'site owner';
+			if(($mybb->settings['contactlink'] == "contact.php" && $mybb->settings['contact'] == 1 && ($mybb->settings['contact_guests'] != 1 && $mybb->user['uid'] == 0 || $mybb->user['uid'] > 0)) || $mybb->settings['contactlink'] != "contact.php")
```

Comment:
` && $mybb->user['uid'] == 0` here seems redundant.

Reply:
This is copied from `global.php`: https://github.com/mybb/mybb/blob/128434e0b038a0b28f9cfc9e0d42bdd5cc28972b/global.php#L1038

Reply:
Still redundant but the main reason to ask for changes was actually what I mentioned above.

> Also, maybe you can check THIS_SCRIPT against contact.php to display the admin email instead of the contact link.

I'm fine with everything else. If users aren't able to use the core contact form maybe we should display something else like the contact email or simply no link.

Reply:
Yes, makes sense. Will change it, but I don’t think we should expose the contact email address itself - will probably just not have a contact link.